index.html

```diff
@@ -0,0 +1,78 @@
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="top-nav">
+        <a href="#" class="logo-txt"> Makoji David </a>
+        <div class="menu">
+          <img src="./img/icons/hamburger.png" alt="logo-icon" class="logo-icon" />
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="jumbotron">
+        <h1 class="title">Onate Makoji David</h1>
+        <h3 class="subtitle">Software developer</h3>
+        <p class="description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <hr />
+      </section>
+      <div class="links">
+        <div class="menu-list">
+          <ul class="lists biggi">
+            <li class="label-txt">Languages</li>
+            <li class="list-item"><a href="#" class="list-links">Javascript</a></li>
+            <li class="list-item"><a href="#" class="list-links">Ruby</a></li>
+            <li class="list-item"><a href="#" class="list-links">Html</a></li>
+            <li class="list-item"><a href="#" class="list-links">Css</a></li>
+          </ul>
+          <ul class="lists biggi">
+            <li class="label-txt">Frameworks</li>
+            <li class="list-item"><a href="#" class="list-links">Bootstrap</a></li>
+            <li class="list-item"><a href="#" class="list-links">Ruby on Rails</a></li>
+            <li class="list-item"><a href="#" class="list-links">RSPec</a></li>
+            <li class="list-item"><a href="#" class="list-links">CapyBara</a></li>
+            <li class="list-item"><a href="#" class="list-links">Selenium</a></li>
+          </ul>
+          <ul class="lists biggi">
+            <li class="label-txt">Skills</li>
+            <li class="list-item"><a href="#" class="list-links">Codekit</a></li>
+            <li class="list-item"><a href="#" class="list-links">Github</a></li>
+            <li class="list-item"><a href="#" class="list-links">Codepen</a></li>
+            <li class="list-item"><a href="#" class="list-links">Gitlab</a></li>
+            <li class="list-item"><a href="#" class="list-links">Terminal</a></li>
+          </ul>
+        </div>
+        <hr />
+      </div>
+      <div class="social-icons">
+        <ul class="social-links">
+          <li>
+            <a href="#"><img src="./img/icons/github.svg" alt="github-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/linkedin.svg" alt="linkedIn-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/angellist.svg" alt="angellist-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/twitter.svg" alt="github-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/medium.svg" alt="github-icon" /></a>
```

Comment:
- [ ] [OPTIONAL] Please make sure the color of these social icons should resemble the Figma design. You can change the fill color of SVG files. 
- [ ] [OPTIONAL] Please make sure the layout of the list of technologies looks like the Figma design. 
- [ ] [OPTIONAL] Kindly add the `Work` button including the arrow icon before it. 

<p align="center">&nbsp;
 
| Your project |   Figma design |
| ---------- | ----------- |
 | ![socialicons](https://user-images.githubusercontent.com/12524453/176557355-91c6fd52-2003-4fe6-af31-fc9175312d6c.png) | ![Template 4-main-mobile(1)](https://user-images.githubusercontent.com/12524453/176557515-569ad8cd-e077-4e0c-b33c-3a68873d053a.png) |
</p>